client/index.html

```diff
@@ -241,29 +241,28 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Notifications</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 								<input id="badge" type="checkbox" name="badge">
 								Enable badge
 								</label>
-							</div>
-							<div class="col-sm-12">
```

Comment:
So, I've changed slightly how the settings page is organised, because otherwise having an input with a label ended up making the other side have a blank line. 

Will post screenshots later.